src/components/ADempiere/Form/VPOS/Collection/overdrawnInvoice/index.vue

```diff
@@ -0,0 +1,278 @@
+<!--
+ ADempiere-Vue (Frontend) for ADempiere ERP & CRM Smart Business Solution
+ Copyright (C) 2017-Present E.R.P. Consultores y Asociados, C.A.
+ Contributor(s): Elsio Sanchez elsiosanches@gmail.com www.erpya.com
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or
+ (at your option) any later version.
+
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License
+ along with this program.  If not, see <https:www.gnu.org/licenses/>.
+-->
+<template>
+  <div>
+    <el-dialog
+      :title="$t('form.pos.collect.overdrawnInvoice.title')"
+      :visible.sync="showDialogo"
+      width="70%"
+      @close="close"
+    >
+      <div v-if="caseOrder === 1">
+        <el-form>
+          <el-form-item>
+            <el-radio v-model="option" :label="1"> {{ $t('form.pos.collect.overdrawnInvoice.returned') }} {{ formatPrice(change, currency.iSOCode) }} </el-radio>
+            <el-radio v-model="option" :label="2"> {{ $t('form.pos.collect.overdrawnInvoice.coupon') }}</el-radio>
+            <el-radio v-model="option" :label="3"> {{ $t('form.pos.collect.overdrawnInvoice.returnMoney') }}</el-radio>
+            <el-radio v-model="option" :label="4"> {{ $t('form.pos.collect.overdrawnInvoice.adjustDocument') }}</el-radio>
+          </el-form-item>
+        </el-form>
+        <el-card v-if="option === 2" class="box-card">
+          <div slot="header" class="clearfix">
+            <span>{{ $t('form.pos.collect.overdrawnInvoice.above') }}</span>
+          </div>
+          <div class="text item">
+            <el-form
+              label-position="top"
+              label-width="10px"
+            >
+              <el-row>
+                <el-col v-for="(field, index) in fieldsList" :key="index" :span="8">
```

Comment:
create a computed and evaluate the column conditions there returning the required list of fields to be looped through with v-for in a `template` tag since the `el-col` tag is inside the `field-definintion` component, example:
```javascript
fieldsListAbove() {
  return this.fieldsList.filter(fieldItem => {
    ['Name', 'Phone', 'Vaue'].includes(fieldItem.columnName)
  }
}
```

and remove the v-if from field-definition